frontend/public/components/RBAC/bindings.jsx

```diff
@@ -81,9 +81,7 @@ const menuActions = ({ subjectIndex, subjects }, startImpersonate) => {
 
   const actions = [
     (kind, obj) => ({
-      label: i18next.t('public~Duplicate {{kindLabel}}', {
-        kindLabel: getKindLabel(kind),
-      }),
+      label: i18next.t('public~Duplicate RoleBinding'),
```

Comment:
This will result in the label for non-RoleBindings being labeled "RoleBinding", no?

Reply:
Would this action have a label other than "RoleBinding"?